godotopenxrmeta/src/main/cpp/openxr_fb_spatial_entity_container_extension_wrapper.cpp

```diff
@@ -0,0 +1,109 @@
+#include <godot_cpp/classes/object.hpp>
+
+using namespace godot;
+
+OpenXRFbSpatialEntityContainerExtensionWrapper *OpenXRFbSpatialEntityContainerExtensionWrapper::singleton = nullptr;
+
+OpenXRFbSpatialEntityContainerExtensionWrapper *OpenXRFbSpatialEntityContainerExtensionWrapper::get_singleton() {
+	if (singleton == nullptr) {
+		singleton = memnew(OpenXRFbSpatialEntityContainerExtensionWrapper());
+	}
+	return singleton;
+}
+
+OpenXRFbSpatialEntityContainerExtensionWrapper::OpenXRFbSpatialEntityContainerExtensionWrapper() :
+		OpenXRExtensionWrapperExtension() {
+
+	ERR_FAIL_COND_MSG(singleton != nullptr, "An OpenXRFbSpatialEntityContainerExtensionWrapper singleton already exists.");
+
+	request_extensions[XR_FB_SPATIAL_ENTITY_CONTAINER_EXTENSION_NAME] = &fb_spatial_entity_container_ext;
+	singleton = this;
+}
+
+OpenXRFbSpatialEntityContainerExtensionWrapper::~OpenXRFbSpatialEntityContainerExtensionWrapper() {
+	cleanup();
+}
+
+void OpenXRFbSpatialEntityContainerExtensionWrapper::_bind_methods() {
+	ClassDB::bind_static_method("OpenXRFbSpatialEntityContainerExtensionWrapper", D_METHOD("get_singleton"), &OpenXRFbSpatialEntityContainerExtensionWrapper::get_singleton);
+	ClassDB::bind_method(D_METHOD("is_spatial_entity_container_supported"), &OpenXRFbSpatialEntityContainerExtensionWrapper::is_spatial_entity_container_supported);
+}
+
+void OpenXRFbSpatialEntityContainerExtensionWrapper::cleanup() {
+	fb_spatial_entity_container_ext = false;
+}
+
+godot::Dictionary OpenXRFbSpatialEntityContainerExtensionWrapper::_get_requested_extensions() {
+	godot::Dictionary result;
+	for (auto ext: request_extensions) {
+		godot::String key = ext.first;
+		uint64_t value = reinterpret_cast<uint64_t>(ext.second);
+		result[key] = (godot::Variant)value;
+	}
+	return result;
+}
+
+void OpenXRFbSpatialEntityContainerExtensionWrapper::_on_instance_created(uint64_t instance) {
+	if (fb_spatial_entity_container_ext) {
+		bool result = initialize_fb_spatial_entity_container_extension((XrInstance)instance);
+		if (!result) {
+			UtilityFunctions::print("Failed to initialize fb_spatial_entity_container extension");
+			fb_spatial_entity_container_ext = false;
+		}
+	}
+}
+
+void OpenXRFbSpatialEntityContainerExtensionWrapper::_on_instance_destroyed() {
+	cleanup();
+}
+
+bool OpenXRFbSpatialEntityContainerExtensionWrapper::initialize_fb_spatial_entity_container_extension(const XrInstance p_instance) {
+	GDEXTENSION_INIT_XR_FUNC_V(xrGetSpaceContainerFB);
+
+	return true;
+}
+
+std::vector<XrUuidEXT> OpenXRFbSpatialEntityContainerExtensionWrapper::get_contained_uuids(XrSpace space) {
+    XrSpaceContainerFB spaceContainer = {XR_TYPE_SPACE_CONTAINER_FB};
+    xrGetSpaceContainerFB(SESSION, space, &spaceContainer);
+    std::vector<XrUuidEXT> uuids(spaceContainer.uuidCountOutput);
+    spaceContainer.uuidCapacityInput = uuids.size();
+    spaceContainer.uuids = uuids.data();
+    xrGetSpaceContainerFB(SESSION, space, &spaceContainer);
+		return uuids;
+}
```

Comment:
nit: the indents used in this method are different from the ones used in the previous methods.